### ISSUE TITLE
Ability to scope components/vulnerabilties

### ISSUE THREAD

**Issue by ScottChapman** [DependencyTrack/dependency-track#927]
When used by multiple teams, it is important to be able to understand what applies to the set of projects they are responsible for. So, being able to establish "scope" results in other views (i.e. component and vulnerability views) to a subset of projects (which might be established by ACL or search or something else)

There are clearly different ways of achieving this so I'll try to be implementation agnostic. But the basic idea is that there are only a subset of the projects I am interested in, so I'd like to only have to look through results that apply to those projects.

**Comment by stevespringett** [comment]
Restricting access to projects based on ACL is planned #140. It's one of my highest priority items and will likely take an entire release to get right.

Also, you can use tags already to group projects together so that you're only seeing the projects you care about. That is not the default view however, nor is it restricted.

**Comment by ScottChapman** [comment]
Right, I saw #140 That probably does help, except for people participating in more than one team I suppose.

Are you saying in the UI I can use tags to search for projects? I didn't see how to do that exactly.
